deployment.go

```diff
@@ -109,6 +110,43 @@ var deploymentStatusCmd = &cobra.Command{
 	Requires the Inertia daemon to be active on your remote - do this by 
 	running 'inertia [REMOTE] up'`,
 	Run: func(cmd *cobra.Command, args []string) {
+		config, err := client.GetProjectConfigFromDisk()
+		if err != nil {
+			log.WithError(err)
+		}
+		remotename := strings.Split(cmd.Parent().Use, " ")[0]
+		remote, found := config.GetRemote(remotename)
+		if !found {
+			println("No such remote " + remotename)
+			println("Inertia currently supports one remote per repository")
+			println("Run `inertia remote -v' to see what remote is available")
+			os.Exit(1)
+		}
+
+		host := "http://" + remote.GetIPAndPort()
+		remoteResp, err := http.Get(host)
+		if err != nil {
+			println("Could not connect to daemon")
+			println("Try running inertia [REMOTE] init")
+			return
+		}
+		defer remoteResp.Body.Close()
+		remoteBody, err := ioutil.ReadAll(remoteResp.Body)
```

Comment:
See my previous comments - all this has essentially been done already by [`deployment.Status()`](https://github.com/ubclaunchpad/inertia/blob/master/client/deployment.go#L85)